checkov/arm/checks/resource/PostgreSQLServerConnectionThrottlingEnabled.py

```diff
@@ -8,7 +8,7 @@ class PostgreSQLServerConnectionThrottlingEnabled(BaseResourceCheck):
     def __init__(self):
         name = "Ensure server parameter 'connection_throttling' is set to 'ON' for PostgreSQL Database Server"
         id = "CKV_AZURE_32"
-        #supported_resources = ['Microsoft.DBforPostgreSQL/servers']
+        # supported_resources = ['Microsoft.DBforPostgreSQL/servers']
```

Comment:
@gruebel Let's delete this line, next line is good

Reply:
sure, will do

Reply:
```suggestion
```